python/lsst/afw/fits/_fits.cc

```diff
@@ -276,8 +276,9 @@ PYBIND11_MODULE(_fits, mod) {
     wrappers.addSignatureDependency("lsst.pex.exceptions");
     wrappers.addSignatureDependency("lsst.daf.base");
     // FIXME: after afw.image pybind wrappers are converted
-    //wrappers.addSignatureDependency("lsst.afw.image"); 
-    wrappers.wrapException<FitsError, lsst::pex::exceptions::IoError>("FitsError", "IoError");
+    //wrappers.addSignatureDependency("lsst.afw.image");
```

Comment:
Why do we think it thinks this line changed?

Reply:
extra space previously at the end.